packages/node_modules/@cerebral/inferno/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
We don't use tsconfig, please look at @cerebral/react for how we run the tests...